docker/Dockerfile

```diff
@@ -16,7 +16,7 @@ WORKDIR $PYSETUP_PATH
 COPY ./poetry.lock ./pyproject.toml ./
 RUN $POETRY_HOME/bin/poetry install --no-dev --no-root
 
-FROM python:3.10.6-slim as production
+FROM python:3.11.1-slim as production
```

Comment:
_This change conflicts with the pyproject.toml versions for python_

Reply:
_https://devguide.python.org/versions/ 

3.10 is in bugfix; same as 3.11--so this change seems on the same level of risk we were previously willing to take with a bugfix-state version._